tools/codeformat.py

```diff
@@ -206,7 +206,7 @@ def batch(cmd, files, N=200):
 
     # Format Python files with black.
     if format_py:
-        command = ["black", "--fast", "--line-length=99"]
+        command = ["black", "--fast", f"--config={TOP}{os.sep}pyproject.toml"]
```

Comment:
I suggest having a variable called `BLACK_CFG` defined at the top of this file.

And use `os.path.join.

Reply:
Perhaps a PYPROJECT var with the complete path then makes more sense as that allows other linters to re-use that.

Reply:
Thanks.

And I've also changed it to use .format instead of f-strings (which we can't rely on, to support older versions of Python 3).